### ISSUE TITLE
C#: #undef is split into "#" and "undef" on separate lines after formatting

### ISSUE THREAD

**Issue by uday0408** [Leopotam/vscode-csharpfixformat#55]
### VsCode version: **1.20.0**

C# preprocessor directive #undef is split into "#" and "undef" on separate lines after formatting. It does not happen with other directives.

## Steps to reproduce

1. Insert line at the top of C# file:

`#undef DEBUG`

2. Format the file

## Expected  behavior

`#undef DEBUG`

## Actual behavior

```
#
undef
```

As workaround I'm using:

```
/* fixformat ignore:start */
#undef DEBUG
/* fixformat ignore:end */
```

**Comment by Leopotam** [comment]
@uday0408 can you check last state of develop branch? is issue fixed for you?

**Comment by Leopotam** [comment]
@uday0408 no feedback, closed as fixed. 0.0.61 version released on marketplace.

**Comment by uday0408** [comment]
I confirm the issue is resolved in 0.0.61 version.
